its/projects/MultiLanguageSupport/src/MultiLanguageSupport/should_be_ignored.xml

```diff
@@ -0,0 +1,8 @@
+<parent><child /></parent>
+<?xml version="1.0" encoding="UTF-8"?> <!--  Noncompliant S1778 -->
+
+<dependency>
+  <groupId>semigroup</groupId>
+  <artifactId>haskell.curry.monoid</artifactId>
```

Comment:
hehe

Reply:
This is a beautiful set equipped with an associative binary operation and an identity element. 🪄